model/host/db.go

```diff
@@ -1523,10 +1523,13 @@ func FindHostsScheduledToStop(ctx context.Context) ([]Host, error) {
 	})
 }
 
+// PreStartThreshold is how long in advance Evergreen can check for hosts that
+// are scheduled to start up soon.
+const PreStartThreshold = 5 * time.Minute
```

Comment:
I know that this was only moved, but why not keep it to something like 2 or 3 minutes considering that sleepSchedulerJob runs every minute. Have you found that it takes a long time to get to all the hosts?

Reply:
The 5 minute thing is a safety mechanism because starting a host takes time, and our goal is for the host to be almost guaranteed (as much as possible) to be up by the time the user schedules it to be up. Given that there might be a slight latency in each step of the process (sleep scheduler -> start job -> actually starting the host) and potential variation in the time to start a host in AWS. There's not really any harm in starting it up a little earlier for reliability purposes.